activemodel/lib/active_model/errors.rb

```diff
@@ -381,6 +381,11 @@ def group_by_attribute
     # If +type+ is a proc, it will be called, allowing for things like
     # <tt>Time.now</tt> to be used within an error.
     #
+    # If +type+ is compatible, +options+ will be injected in the translation.
```

Comment:
Instead of placing this here, I would put it directly under the `If +type+ is a symbol...` line and add an example with and without interpolation like:
```
person.errors.add(:name, :blank)
person.errors.messages
# => {:name=>["can't be blank"]}

person.errors.add(:name, :too_long, { count: 25 })
person.errors.messages
# => ["is too long (maximum is 25 characters)"]
```

Reply:
That's indeed a nicer way to do it, thank !